chunks/lang-elixir/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG base
+FROM ${base}
+
+USER root
+
+# Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
+ENV TRIGGER_REBUILD=1
+
+RUN wget https://packages.erlang-solutions.com/erlang-solutions_2.0_all.deb \
+    && sudo dpkg -i erlang-solutions_2.0_all.deb \
```

Comment:
do we need this `sudo`? User is already set to root on line 4

Reply:
agreed, will push changes